db/indextest/util_testing.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020-Present Couchbase, Inc.
+
+Use of this software is governed by the Business Source License included in
+the file licenses/BSL-Couchbase.txt.  As of the Change Date specified in that
+file, in accordance with the Business Source License, use of this software will
+be governed by the Apache License, Version 2.0, included in the file
+licenses/APL2.txt.
+*/
+
+package indextest
```

Comment:
util_testing.go is generally intended for test functionality that needs to be made available cross-package.  Shared test functionality that's intended for use only within the package can just be defined in one of the test files.

Reply:
I see. I thought it would be a place to have common function defined that are used for testing. I will move these functions into test file.